### PR TITLE
Typo fix in 02.API-REFERENCE.md

### PR DESCRIPTION
**Pull request by batizdaniel**
JerryScript-DCO-1.0-Signed-off-by: Daniel Batiz batizjob@gmail.com

### CODE REVIEW

File: docs/02.API-REFERENCE.md [hunk 1]
```diff
@@ -1495,7 +1497,7 @@ someplace_in_the_code (void)
 
 Registers an external magic string array.
 
-*Notes*:
+*Note*:
```
Comment (akosthekiss) [inline docs/02.API-REFERENCE.md:1500]: My two cents: plural (i.e., "Notes") is quite OK when there are multiple comments.

(Of course, it depends whether "Note" is a verb or a noun here. If it stands for "Note that...", then the trailing "s" is incorrect. If it stands for the noun "Note", then my above comment holds.)

Comment (zherczeg) [reply docs/02.API-REFERENCE.md:1500]: I think it is very easy to forget updateing note / notes after a comment is added / removed. I would prefer to use a single form.
Alternative: `Additional info:` It has no plural problem.

**Review by zherczeg — APPROVED** [review]
LGTM, may need an update

**Comment by LaszloLango** [comment]
Cannot be applied. The master diverged a lot.

**Pull request closed without merging**